src/middlewares/validations/comment.update.delete.js

```diff
@@ -13,7 +13,7 @@ import status from '../../helpers/constants/status.codes';
  * @returns {Function} - A fuction rendering an arror message
  * or the next callback
  */
-export default async (req, res, next) => {
+export default (req, res, next) => {
   // Initializing variables
```

Comment:
Why are removing this async?

Reply:
it was not used anywhere in the code....it was no reason to keep it